server/controllers/users/userController.ts

```diff
@@ -0,0 +1,78 @@
+import { Response } from 'express';
+import Interviewee from '../../database/Models/Interviewee';
+import { RequestType } from '../../utils';
+import validateQuery from '../../utils/validation/queryValidation';
+
+const getAllReviews = async (req: RequestType, res: Response) => {
+  const { page = '1', saved = undefined } = req.query;
+
+  await validateQuery(req.query);
+  // Convert the incoming saved string into a boolean
+  const savedBoolean : any = saved ? JSON.parse(saved.toString())
+    : saved;
+
+  // const id = req.userInfo?.id;
+
+  // filter reviews based on the saved parameter and paginate the results
+  // TODO: Replace the static userId with the logged in user's id when create review is implemented
```

Comment:
Why not to add the id ?

Reply:
I was testing if the function works for a user who already has reviews, if I added the userId right now it will get no reviews at all

Reply:
You can log in with the one you want to test on, we need this to work correctly so we can use it in the front.